core/src/main/scala/org/dbpedia/spotlight/db/tokenize/OpenNLPTokenizer.scala

```diff
@@ -11,11 +11,11 @@ import org.dbpedia.spotlight.db.model.{TokenTypeStore, Stemmer}
  */
 
 class OpenNLPTokenizer(
-  tokenizer: opennlp.tools.tokenize.Tokenizer,
+  tokenizer: opennlp.tools.tokenize.TokenizerME,
   stopWords: Set[String],
   stemmer: Stemmer,
   sentenceDetector: SentenceDetector,
-  var posTagger: POSTagger,
+  var posTagger: POSTaggerME,
```

Comment:
Does anyone know if it is available in a domain the model in English Lucene backend ?
Atenciosamente, 
Laura Lima DiasBacharela em Ciência da Computação, IF Sudeste MG - Campus Rio Pomba
Mestranda em Ciência da Computação, Universidade Federal de Juiz de Fora (UFJF)



Date: Mon, 21 Mar 2016 06:07:49 -0700
From: notifications@github.com
To: dbpedia-spotlight@noreply.github.com
Subject: Re: [dbpedia-spotlight] OpenNLPTokenizer optimization and new features (#400)

In core/src/main/scala/org/dbpedia/spotlight/db/tokenize/OpenNLPTokenizer.scala:

>    stopWords: Set[String],
>    stemmer: Stemmer,
>    sentenceDetector: SentenceDetector,
> -  var posTagger: POSTagger,
> +  var posTagger: POSTaggerME,

 It's fine in that case.


—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub